portal-frontend/src/app/features/applications/edit-submission/proposal/subd-proposal/subd-proposal.component.html

```diff
@@ -264,9 +264,9 @@ <h6>Documents needed for this step:</h6>
         (deleteFile)="onDeleteFile($event)"
         (openFile)="openFile($event)"
         [showErrors]="showErrors"
-        [isRequired]="isHomeSiteSeverance.getRawValue() !== 'false'"
+        [isRequired]="isHomeSiteSeverance.getRawValue() !== 'true'"
         [allowMultiple]="true"
-        [disabled]="isHomeSiteSeverance.getRawValue() === 'false'"
+        [disabled]="isHomeSiteSeverance.getRawValue() !== 'true'"
```

Comment:
just curious what is the reason behind changing from === 'false' to !== 'true'

Reply:
The original requirements had the form field disabled if they chose false, now its disabled if they haven't selected anything or its false. So this check now covers null.